layouts/partials/head.html

```diff
@@ -29,14 +30,12 @@
   crossorigin="anonymous"></script>
 {{ if .Site.Params.offlineSearch }}
 <script
-  src="https://unpkg.com/lunr@2.3.9/lunr.min.js"
-  integrity="sha384-203J0SNzyqHby3iU6hzvzltrWi/M41wOP5Gu+BiJMz5nwKykbkUx8Kp7iti0Lpli"
+  src="https://unpkg.com/lunr@2.3.8/lunr.min.js"
+  integrity="sha384-vRQ9bDyE0Wnu+lMfm57BlYLO0/XauFuKpVsZPs7KEDwYKktWi5+Kz3MP8++DFlRY"
   crossorigin="anonymous"></script>
 {{end}}
 {{ if .Site.Params.prism_syntax_highlighting }}
 <!-- stylesheet for Prism -->
 <link rel="stylesheet" href="{{ "/css/prism.css" | relURL }}"/>
 {{ end }}
-<!-- stylesheet for tabbed pane -->
-<link rel="stylesheet" href="{{ "/css/shortcodes.css" | relURL }}"/>
```

Comment:
@jcabak - by removing this link you're reverting #532. Was this intentional?

Reply:
Investigating further, it seems that this was by mistake, but I'd appreciate a confirmation @jcabak.

Reply:
I've created #694 to track this issue.